api/nakadi-event-bus-api.yaml

```diff
@@ -799,6 +799,112 @@ paths:
+      parameters:
+        - name: subscription_id
+          in: path
+          type: string
+          description: Id of subscription
+          required: true
+      responses:
+        '200':
+          description: Ok
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Cursor'
+        '204':
+          description: |
+            No Content. If the reading from the subscription was not started yet - there is nothing to expose.
+            Will return an empty list of cursors to be consistent with successful scenario.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Cursor'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Client is not authorized to perform this operation
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: Subscription not found
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+        '503':
+          description: Service (temporarily) unavailable
+          schema:
+            $ref: '#/definitions/Problem'
+    put:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        Endpoint for committing offsets of the subscription. The offsets are per partition of EventType. One can use
+        this endpoint to commit the offsets for any of the partitions that are part of this subscription.
+        With every commit the offsets can only increase so it is not possible to commit offset that is lower than
+        already committed offset for the appropriate partition; if such request is made - it will not have any effect
+        on the committed offsets.
+      parameters:
+        - name: subscription_id
+          in: path
+          type: string
+          description: Id of subscription
+          required: true
+        - name: cursors
+          in: body
+          description: Cursors with offsets to commit.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Cursor'
+          required: true
+      responses:
+        '200':
+          description: Offsets were committed
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Client is not authorized to perform this operation
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: Subscription not found
+          schema:
+            $ref: '#/definitions/Problem'
+        '409':
+          description: Conflict. The cursors which were tried to be committed are older than already committed ones.
```

Comment:
But then, how can the client be informed that his commit updated server state or not?